cmd/root.go

```diff
@@ -31,12 +30,15 @@ func NewMethodK8s(version string) *MethodK8s {
 	methodK8s := MethodK8s{
 		version: version,
 		RootFlags: config.RootFlags{
```

Comment:
These flags are going to get a bit messy if we just keep adding everything to RootFlags. It might be worth thinking about how you would want to logically group config values together and have structs here.